src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -13,21 +13,254 @@
-	<Thickness x:Key="MaterialButtonBorderThickness">0</Thickness>
-	<Thickness x:Key="MaterialOutlinedButtonBorderThickness">1</Thickness>
-	<CornerRadius x:Key="MaterialButtonCornerRadius">20</CornerRadius>
-	<x:String x:Key="MaterialButtonVerticalContentAlignment">Center</x:String>
-	<x:String x:Key="MaterialButtonHorizontalContentAlignment">Center</x:String>
-
+	<x:Double x:Key="ButtonMinHeight">40</x:Double>
+	<x:Double x:Key="ButtonMinWidth">40</x:Double>
+	<x:Double x:Key="ButtonIconMinWidth">18</x:Double>
+	<x:Double x:Key="ElevatedButtonElevation">1</x:Double>
+	<x:Double x:Key="ElevatedButtonElevationDisabled">0</x:Double>
+	<x:Double x:Key="ButtonElevation">0</x:Double>
+	<x:Double x:Key="IconButtonOpacityVisibleState">1</x:Double>
+	<x:Double x:Key="IconButtonOpacityHiddenState">0</x:Double>
+	<Thickness x:Key="ButtonPadding">16,0</Thickness>
+	<Thickness x:Key="TextButtonPadding">12,0</Thickness>
+	<Thickness x:Key="TextButtonIconMargin">0,0,8,0</Thickness>
+	<Thickness x:Key="ButtonBorderThickness">0</Thickness>
+	<Thickness x:Key="OutlinedButtonBorderThickness">1</Thickness>
+	<Thickness x:Key="ElevatedButtonMargin">0,0,0,1</Thickness>
+	<Thickness x:Key="ElevatedButtonDisabledMargin">0</Thickness>
+	<Thickness x:Key="ButtonContentMargin">8,0</Thickness>
+	<Thickness x:Key="ButtonMargin">0</Thickness>
+	<CornerRadius x:Key="ButtonCornerRadius">20</CornerRadius>
+	<x:String x:Key="ButtonVerticalContentAlignment">Center</x:String>
+	<x:String x:Key="ButtonHorizontalContentAlignment">Center</x:String>
+	<x:String x:Key="ButtonIconHorizontalAlignment">Stretch</x:String>
+	<x:String x:Key="ButtonIconVerticalAlignment">Stretch</x:String>
+	<x:String x:Key="IconButtonEllipseVerticalAlignment">Stretch</x:String>
+	<x:String x:Key="IconButtonEllipseHorizontalAlignment">Stretch</x:String>
+	<x:Boolean x:Key="ElevatedButtonIsTintEnabled">True</x:Boolean>
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<!--MaterialElevatedButtonStyle-->
+			<StaticResource x:Key="ElevatedButtonForeground"
+							ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="ElevatedButtonBackground"
+							ResourceKey="SurfaceBrush" />
+			<StaticResource x:Key="ElevatedButtonBackgroundPointerOver"
+							ResourceKey="PrimaryHoverBrush" />
+			<StaticResource x:Key="ElevatedButtonBackgroundPressed"
+							ResourceKey="PrimaryPressedBrush" />
+			<StaticResource x:Key="ElevatedButtonBackgroundDisabled"
+							ResourceKey="OnSurfaceDisabledLowBrush" />
+			<StaticResource x:Key="ElevatedButtonIconForegroundDisabled"
+							ResourceKey="OnSurfaceDisabledBrush" />
+			<StaticResource x:Key="ElevatedButtonContentForegroundDisabled"
+							ResourceKey="OnSurfaceDisabledBrush" />
+			<StaticResource x:Key="ElevatedButtonBackgroundFocused"
+							ResourceKey="PrimaryFocusedBrush" />
+			<StaticResource x:Key="ElevatedButtonBackgroundPointerFocused"
+							ResourceKey="PrimaryFocusedBrush" />
```

Comment:
We should group the resources for each style by property. So all the "ElevatedButtonBackground[VISUAL_STATE]" brushes should be grouped together before the ButtonIcon and ButtonContent brushes. Same goes for the other button style resources below

Reply:
fixed